### PR TITLE
mat2: fix nautilus-extension

### LINKED ISSUE #157514: mat2: nautilus-plugin not working
### Describe the bug
When starting `nautilus`:
```
Using python version sys.version_info(major=3, minor=9, micro=6, releaselevel='final', serial=0)
Traceback (most recent call last):
  File "/run/current-system/sw/share/nautilus-python/extensions/mat2.py", line 27, in <module>
    from libmat2 import parser_factory
  File "/nix/store/7am33cqjj4zgmr9gbfiivqw27hldwhz6-python3.9-mat2-0.12.2/lib/python3.9/site-packages/libmat2/parser_factory.py", line 31, in <module>
    __load_all_parsers()
  File "/nix/store/7am33cqjj4zgmr9gbfiivqw27hldwhz6-python3.9-mat2-0.12.2/lib/python3.9/site-packages/libmat2/parser_factory.py", line 28, in __load_all_parsers
    importlib.import_module('.' + name, package='libmat2')
  File "/nix/store/2jk1l6lvp9hys6p7lnzhpxdx8hlrdv7q-python3-3.9.6-env/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/nix/store/7am33cqjj4zgmr9gbfiivqw27hldwhz6-python3.9-mat2-0.12.2/lib/python3.9/site-packages/libmat2/audio.py", line 7, in <module>
    import mutagen
ModuleNotFoundError: No module named 'mutagen'
```

### Steps To Reproduce
1. start nautilus on console

### Expected behavior
No error and the context-menu is shown

### Notify maintainers

@dotlambda 
maybe @jtojnar could help?

### Metadata
```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
  /nix/store/q6ayjwizl0xcbvknkcksy5g4n1ipa6b1-bash-interactive-5.1-p8-dev
copying path '/nix/store/q6ayjwizl0xcbvknkcksy5g4n1ipa6b1-bash-interactive-5.1-p8-dev' from 'https://cache.nixos.org'...
 - system: `"x86_64-linux"`
 - host os: `Linux 5.16.2, NixOS, 21.11 (Porcupine)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.16`
 - channels(felix): `""`
 - channels(root): `"nixos-21.11.335526.6c4b9f1a2fd, nixos-unstable-22.05pre343321.78cd22c1b86"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

### PR DESCRIPTION
**Pull request by Stunkymonkey**
###### Motivation for this change
closes #157514
in my setup the entry is shown two times. Might be related to my setup.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/mat2/default.nix [hunk 1]
```diff
@@ -47,6 +47,11 @@ buildPythonPackage rec {
     ./executable-name.patch
     # hardcode path to mat2 executable
     ./tests.patch
+    # fix gmenu path
```
Comment (jtojnar) [inline pkgs/development/python-modules/mat2/default.nix:50]: Comments should be updated to something “Fix gobject-introspection typelib path for Nautilus extension”

**Comment by Stunkymonkey** [comment]
Result of `nixpkgs-review pr 157544` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>mat2 (python39Packages.mat2)</li>
    <li>metadata-cleaner</li>
    <li>python310Packages.mat2</li>
  </ul>
</details>

**Comment by Stunkymonkey** [comment]
ping @dotlambda

**Comment by dotlambda** [comment]
How much does this add to mat2's closure? (I forgot which commands tell you that.)

**Comment by Stunkymonkey** [comment]
i never used it before, and usure if that is what you want:
```
$ nix store diff-closures  /nix/store/hqyidcdjpfgfqx8s70qhm2s6hjbm6966-python3.9-mat2-0.12.3/bin/mat2 /nix/store/5y2bfb50pd0izvcmh9hdmckr74l4dmz6-python3.9-mat2-0.12.3/bin/mat2
glibc: 2.33-108 → 2.33-62, -157.0 KiB
gnutls: 3.7.3 → 3.7.2, -35.9 KiB
util-linux: 2.37.3 → 2.37.2
```

**Comment by jtojnar** [comment]
The command is `nix path-info -Sh`.

**Comment by Stunkymonkey** [comment]
master:
```
$ nix path-info -Sh /nix/store/5y2bfb50pd0izvcmh9hdmckr74l4dmz6-python3.9-mat2-0.12.3/bin/mat2
/nix/store/5y2bfb50pd0izvcmh9hdmckr74l4dmz6-python3.9-mat2-0.12.3	 537.3M
```
this PR:
```
$ nix path-info -Sh /nix/store/hqyidcdjpfgfqx8s70qhm2s6hjbm6966-python3.9-mat2-0.12.3/bin/mat2
/nix/store/hqyidcdjpfgfqx8s70qhm2s6hjbm6966-python3.9-mat2-0.12.3	 537.5M
```

**Review by dotlambda — APPROVED** [review]

**Pull request merged**